src/examples/Examples.js

```diff
@@ -90,6 +92,25 @@ var Examples = React.createClass({
           justify="end" />
         <Box pad={{horizontal: 'large'}}><p><strong>Grommet-only Marquee (no parallax, responsiveness)</strong></p></Box>
         {this._loremIpsum()}
+        <Box pad={{ horizontal: 'large', vertical: 'none' }}>
+          <Callout
+            thumbnail="url(/docs/img/Video_image.png)"
+            label="VIDEO - 4:27"
+            heading="Foundation Paraguay Empowers Microbusinesses"
+            content="See how Hewlett Packard Enterprise delivers mobile solutions to improve quality of life and help eliminate poverty in South America."
+            link="#"
+            linkIcon={<PlayIcon />}
+            linkText="Watch Now"
+          />
+          <Callout
+            thumbnail="url(/docs/img/Case_Study_image.png)"
+            label="CASE STUDY"
```

Comment:
Can you please check with the designers to make sure that this is always going to be in uppercase? If so, we should set this restriction in CSS.